src/strategies/contract-call/examples.json

```diff
@@ -2,20 +2,20 @@
   {
     "name": "Example query",
     "strategy": {
-      "name": "contract-call",
```

Comment:
Looks good, could you just remove change on this file and run the lint script?

Reply:
Done